mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(number) {
+  return number + number / 5
```

Comment:
this is good.
however, there are 2 other better and more readable ways of writing the same:
```js 
// 1:
return number + number * 0.2;

// 2:
return number * 1.2;
```
why these are better? 
coz 0.2 is easier to link with 20% than 1/5; 
what if you are required to add 75% on top of the number?  
0.75 === 3/4 what's better for that?

Reply:
Alright, that's changed, of course it will be easier to use decimals :)